tests/unit/gaze/_utils/_parsing_test.py

```diff
@@ -0,0 +1,920 @@
+# Copyright (c) 2023-2025 The pymovements Project Authors
```

Comment:
I think the filename should be `tests/unit/gaze/_utils/parsing_test.py` or does this conflict with `tests/units/utils/parsing_test.py`?

Reply:
We can also use `gaze/_utils/_parsing.py` or `gaze/_parsing.py` for the source module if you like
As it is a private module, we can change the name anytime.

I wouldn't be much in favor of `gaze/utils/_parsing.py`.